start_esp/nginx-auto.conf.template

```diff
@@ -48,10 +48,7 @@ http {
 % endif
 % endfor
 
-  set_real_ip_from  0.0.0.0/0;
-  set_real_ip_from  0::/0;
-  real_ip_header    X-Forwarded-For;
-  real_ip_recursive on;
```

Comment:
ideally, we should support both,  check x-user-ip first, then xff.   users should not need to config it.